docs/getting-started/github-apps-troubleshooting-tips.rst

```diff
@@ -0,0 +1,14 @@
+.. Need to update with info about checking lambda errors in UI
+
+- If you don't see changes, try waiting a couple of minutes and refreshing the page again.
+- Verify the GitHub app was given access to the right repository. If access was given to the wrong repo, you'll need to make another push after correcting it. Check by:
+
+    - Navigating to the ``+ Manage Dockstore Installation on GitHub`` button mentioned above
+    - Go to your repo on GitHub, click the Settings tab, click Integrations on the left and verify our app is installed and configured correctly
```

Comment:
I'd add screenshots for this, both from our page and on Github. It's not that easy to find.